models/models.go

```diff
@@ -129,7 +129,7 @@ func (instance *Instance) validateInstance(ctx context.Context, isCantabularFlex
 }
 
 // validateCodelists - checks if fields of CodeList are not empty for ValidateAddRecipe, ValidateAddInstance, ValidateAddCodelist
-func (c *CodeList) validateCodelist(ctx context.Context, isCantabularFlexibleTable bool) (missingFields []string, invalidFields []string) {
+func (c *CodeList) validateCodelist(ctx context.Context, isCantabularFlexibleTable, isCantabularType bool) (missingFields []string, invalidFields []string) {
```

Comment:
similar thing here i would probably pass in the recipe, to avoid method signature pollution
```suggestion
func (c *CodeList) validateCodelist(ctx context.Context, recipe *models.Recipe) (missingFields []string, invalidFields []string) {
```